### PR TITLE
Ansible guide improvements

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Contribution Guide
+
+The infrastructure project contains:
+
+1. The [Ansible](ansible) scripts for bootstrapping the build and test hosts (including a way to test Ansible)
```
Comment (neomatrix369) [inline CONTRIBUTING.md]: Minor nit - the Ansible link goes to the Ansible guide, but the text says Ansible scripts, might be best to point to the folder that contains the scripts

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+To make a change please:
+ 
+1. Fork this repository
+1. Create a branch off your fork
+1. Make the change
+1. Test it (see below)
+1. Submit a Pull Request
+
+Only reviewers in the `admin_infrastructure` team have permission to merge requests for this `openjdk-infrastructure` repo,
```
Comment (neomatrix369) [inline CONTRIBUTING.md]: Might be worth showing a link to the reviewers, it is possible to create a reviewers group on GitHub, and link to it?

File: ansible/README.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
 # Ansible playbooks to download and install dependencies for OpenJDK on various platforms
 
+# Running via Vagrant and VirtualBox 
+
+You are better off running it inside a Virtual Machine isolate from your own local system.
```
Comment (neomatrix369) [inline ansible/README.md]: isolated

**Comment by karianna** [comment]
The zulu addition fails on the test Vagrant with:
`fatal: [localhost]: FAILED! => {"failed": true, "msg": "An unhandled exception occurred while running the lookup plugin 'file'. Error was a <class 'ansible.errors.AnsibleError'>, original message: could not locate file in lookup: /Vendor_Files/keys/id_rsa.pub"}`)`

File: ansible/README.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+**NOTE** The `/vagrant/` directory maps to the directory on your host that you launched the `VagrantFile` from 
+e.g. `/home/karianna/workspace/AdoptOpenJDK/openjdk-infrastructure/ansible`
+
+```
+   $ vagrant up
+   $ vagrant ssh
+   
+   (uses default ssh login: vagrant   password: vagrant)
+   
+   cd /vagrant/playbooks
```
Comment (neomatrix369) [inline ansible/README.md]: :+1:

**Comment by neomatrix369** [comment]
Azul ppa fails for me as well not related to this PR though

File: ansible/README.md [hunk 5]
```diff
@@ -1,5 +1,31 @@
...
+   
+   cd /vagrant/playbooks   
+```
+
+1) Ensure that you have edited `hosts` file in `/etc/ansible/`. 
+For running locally `hosts` file should contain something as simple as `localhost ansible_connection=local`.
+
+2) Run a playbook to install dependencies, for Ubuntu 14.x on x86:
+
+`ansible-playbook -s ubuntu.yml`
```
Comment (neomatrix369) [inline ansible/README.md]: :+1: works

File: ansible/playbooks/ubuntu.yml [hunk 6]
```diff
@@ -33,6 +33,13 @@
       apt_repository: repo='deb http://repos.azulsystems.com/ubuntu stable main'
       tags: patch_update
 
+    - name: Add the azul key
+      apt_key:
```
Comment (neomatrix369) [inline ansible/playbooks/ubuntu.yml]: :exclamation: Secrets alert! Is there some way to store this info on vault or some secret place and read it off from there? Does ansible provide such facility?

**Comment by karianna** [comment]
@neomatrix369 we should try to resolve the:

```
fatal: [localhost]: FAILED! => {"failed": true, "msg": "An unhandled exception occurred while running the lookup plugin 'file'. Error was a <class 'ansible.errors.AnsibleError'>, original message: could not locate file in lookup: /Vendor_Files/keys/id_rsa.pub"})
``` 

issue, so we've got a confirmed test in the Vagrant'd test host.

**Comment by neomatrix369** [comment]
@karianna I did get this as well, yes lets try to resolve this before we merge the PR

**Comment by neomatrix369** [comment]
@karianna did you have any luck finding out what the issue is?

**Comment by karianna** [comment]
@neomatrix369 Not yet no sorry, let's discuss in Slack

**Comment by karianna** [comment]
@neomatrix369 I think this is good to go again as this is just doc changes now.  We can raise a separate issue for the failing test ansible vagrant thing

**Comment by neomatrix369** [comment]
In https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/58#issuecomment-347155454, you mean the vagrant instance is our test host?

https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/58#issuecomment-347463749 - Agree, lets rasie a separate issue for this

**Comment by karianna** [comment]
Yes vagrant as test image - are you able to set this review to approved?

**Comment by neomatrix369** [comment]
@karianna I don't have rights to approval/merge PRs, will merge PR is added to reviewers list

**Pull request merged**